tests/e2e/e2e_test.go

```diff
@@ -1152,10 +1198,103 @@ func TestE2E(t *testing.T) {
 				if err != nil {
 					return err
 				}
-
 				return removeNodeTaint(t, f, taintedNode.Name, taintKey)
 			},
 		},
+		testExecution{
+			Name:       "TestScanSettingBinding",
+			IsParallel: true,
+			TestFn: func(t *testing.T, f *framework.Framework, ctx *framework.Context, mcTctx *mcTestCtx, namespace string) error {
+				objName := getObjNameFromTest(t)
+
+				rhcosPbName := objName + "-rhcos4"
+				err := createReadyProfileBundle(t, f, rhcosPbName, namespace, contentImagePath, rhcosContentFile)
+				if err != nil {
+					return err
+				}
+
+				rhcos4e8profile := &compv1alpha1.Profile{}
+				key := types.NamespacedName{Namespace:namespace, Name: rhcosPbName + "-e8"}
+				err = f.Client.Get(goctx.TODO(), key, rhcos4e8profile)
+				if err != nil {
+					return err
+				}
+
+				scanSettingName := objName + "-setting"
+				scanSetting := compv1alpha1.ScanSetting{
+					ObjectMeta:            metav1.ObjectMeta{
+						Name:      scanSettingName,
+						Namespace: namespace,
+					},
+					ComplianceSuiteSettings: compv1alpha1.ComplianceSuiteSettings{
+						AutoApplyRemediations: false,
+					},
+					ComplianceScanSettings: compv1alpha1.ComplianceScanSettings{
+						Debug: true,
+					},
+					Roles:                 []string{"master", "worker"},
+				}
+
+				err = f.Client.Create(goctx.TODO(), &scanSetting, nil)
+				if err != nil {
+					return err
+				}
+
+				scanSettingBindingName := "generated-suite"
+				scanSettingBinding := compv1alpha1.ScanSettingBinding{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      scanSettingBindingName,
+						Namespace: namespace,
+					},
+					Profiles:   []compv1alpha1.NamedObjectReference{
+						// TODO: test also OCP profile when it works completely
```

Comment:
the OCP profile is not working?

Reply:
Wasn't there a case where you couldn't use the whole OCP profile because of some issue in the content? I forgot the details, but we were only selecting a single rule because of this...

Reply:
It was an issue with the expectations of the test code. Not the profile itself.